### PR TITLE
7470 add shipping category feature test

### PR DESCRIPTION
**Pull request by shen-sat**
#### What? Why?
This PR adds a feature test for adding shipping categories 
Part of #7470 

Changelog Category: Technical changes

### CODE REVIEW

File: app/views/spree/admin/shipping_categories/new.html.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - content_for :page_actions do
   %li
-    = button_link_to Spree.t(:back_to_shipping_categories_list), spree.admin_shipping_categories_path, :icon => 'icon-arrow-left'
+    = button_link_to Spree.t(:back_to_shipping_categories), spree.admin_shipping_categories_path, :icon => 'icon-arrow-left'
```
Comment (shen-sat) [inline app/views/spree/admin/shipping_categories/new.html.haml:8]: Test had been failing due to a `missing translation key` so I fixed it here

**Review by andrewpbrett — APPROVED** [review]
This looks great! @filipefurtad0 should probably have a look to see if it's sufficient to close #7470 but regardless this is a good addition.

File: app/views/spree/admin/shipping_categories/new.html.haml (hunk 1 above)
Comment (mkllnk) [reply app/views/spree/admin/shipping_categories/new.html.haml:8]: Great. If you had done this in a separate commit then you could have added the explanation there. Everyone looking through the codes history can then see this context.

File: spec/features/admin/configuration/shipping_categories_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "Tax Categories" do
+  include AuthenticationHelper
+  # include WebHelper
```
Comment (mkllnk) [inline spec/features/admin/configuration/shipping_categories_spec.rb:7]: If you don't need the WebHelper, you can remove this line completely.

**Review by mkllnk — CHANGES_REQUESTED** [review]
Great work!

I'm only moving this back to In-Dev to correct your typo (see Andy's comment) and the superfluous line (WebHelper).

File: app/views/spree/admin/shipping_categories/new.html.haml (hunk 1 above)
Comment (shen-sat) [reply app/views/spree/admin/shipping_categories/new.html.haml:8]: Thanks for the tip @mkllnk, I should use my commit messages to express myself more 👍

File: spec/features/admin/configuration/shipping_categories_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "Tax Categories" do
```
[the remark this answers is not in the document]
Comment (shen-sat) [reply spec/features/admin/configuration/shipping_categories_spec.rb:5]: Ah whoops 🙈 I'll change now 👍

**Review by mkllnk — APPROVED** [review]

**Comment by filipefurtad0** [comment]
I think having these controller tests is sufficient to cover that page functionality :+1: 
Awesome @shen-sat, thank you !

**Comment by filipefurtad0** [comment]
Awesome @shen-sat,
I think this closes #7470, as you've added the controller test before as well on #7686. The bit missing was the interaction on create/edit which is covered here, so I think we're good :+1:  
Thanks!

**Pull request merged**